Analytics/Classes/SEGAnalytics.m

```diff
@@ -176,6 +178,20 @@ - (void)_applicationWillEnterForeground
     }];
 }
 
+- (void)_applicationDidEnterBackground
+{
+  if (!self.configuration.trackApplicationLifecycleEvents) {
+    return;
+  }
+  NSString *currentVersion = [[NSBundle mainBundle] infoDictionary][@"CFBundleShortVersionString"];
+  NSString *currentBuild = [[NSBundle mainBundle] infoDictionary][@"CFBundleVersion"];
+  [self track: @"Application Backgrounded" properties: @{
```

Comment:
I don't believe any properties should be collected here, as per https://segment.com/docs/spec/mobile/#application-backgrounded.